MREUnityRuntime/MREUnityRuntimeLib/Behaviors/Handlers/PenHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using MixedRealityExtension.API;
+using MixedRealityExtension.App;
+using MixedRealityExtension.Behaviors.ActionData;
+using MixedRealityExtension.Behaviors.Actions;
+using MixedRealityExtension.Core.Interfaces;
+using MixedRealityExtension.PluginInterfaces.Behaviors;
+using System;
+
+namespace MixedRealityExtension.Behaviors.Handlers
+{
+	internal class PenToolHandler : ToolHandler<PenToolData>
```

Comment:
>PenToolHandler [](start = 16, length = 14)

Rename this file to match class name?

Reply:
Or, rename the class to PenHandler.

---
In reply to: [384833386](https://github.com/microsoft/mixed-reality-extension-unity/pull/123#discussion_r384833386) [](ancestors = 384833386)

Reply:
Good catch.  I had renamed the file but not the class.